pynbody/plot/sph.py

```diff
@@ -464,13 +471,29 @@ def image(sim, qty='rho', width="10 kpc", resolution=500, units=None, log=True,
             units = im.units
 
 
-        if units.latex() is "":
-            units=""
-        else:
-            units = "$"+units.latex()+"$"
-
+        # RS - Fix formatting.
         if show_cbar:
-             plt.colorbar(ims).set_label(qtytitle+"/"+units)
+            if log:
```

Comment:
Use log10(value) in the colorbar if the colorbar is on a log scale... saves space and agrees with labels like "log \rho"